data/projects/implement_aspects.md

```diff
@@ -1,30 +1,70 @@
-In the future, coala will choose the bears to run for the user based on what
-aspects of a programming language he wants to analyze. This project should
-realize this.
+Initial configuration of coala is **unnecessary hard**, especially for new
+users. There are a few reasons for this.
 
-The aspects concept exists as
-[cEP-0005](https://github.com/coala/cEPs/blob/master/cEP-0005.md) and it is
+New users are not always familiar with all the functionality provided by various
+tools inside coala. Besides that, coala tries to provide a full set of options
+for those tools, which could make configuring each tools overwhelming.
+Finally, there is the possibility that the same type of checking could be
+executed by multiple tools, which makes choosing the right tools for a type of
+analysis hard.
+
+Right now in coala, if users want to check their code, they need to know what
+existing tools (or in coala's terms, bears) have the capability to do that.
+For example, they want to check code duplication in Ruby code. RuboCopBear and
+RubySmellBear both could do that. The user needs to chose one of them and then
+configure it to check code duplication.
+
+This is where "aspects" do their work. We could say "aspects" is what kind of
+analysis we want to run over the code. For example, "CodeCloneDetection" is a
+kind of aspects that will detect a portion of code, that have exact duplicates
+somewhere in the program and warn the user about it. Now if a user wants to
+detect duplicated code, instead of stating which bear coala must use
+(which requires the user to know bears/tools and their options),
+they can simply add the CodeCloneDetection aspect and coala will figure out
+what bears must be used and how to use them.
+
+Further technical explanation about aspects is explained in
+[cEP-0005](https://github.com/coala/cEPs/blob/master/cEP-0005.md). It is
 already possible to define aspects and use them in bears - however this has no
 effect because coala doesn't understand what to do with them yet.
 
-This project is about implementing aspects support in coala. coala will have to
-choose the right bears on the basis of aspects and show the right results.
-It should be able to ignore aspects just like bear ignoring works right now.
+This project's goal is to implement aspects support in coala.
+
+This means coala will have to choose the right bears on the basis of aspects
+and show the right results. It should also be able to ignore aspects just like
+bear ignoring works right now.
+
+The difference between this project and "Convert Bears to Aspects" is that the
+other project's focuses on defining aspects on *each bears* while this project
```

Comment:
darn :D I missed the or on this part. project's -> project
then we merge this :D